boards/splitkb/aurora/lily58/main.py

```diff
@@ -0,0 +1,99 @@
+from kb import KMKKeyboard
+from kmk.modules.split import Split
+from kmk.modules.layers import Layers
+from kmk.modules.holdtap import HoldTap
+from kmk.extensions.rgb import RGB
+from kmk.extensions.peg_oled_display import Oled,OledDisplayMode,OledReactionType,OledData
```

Comment:
The use `peg_oled_display` is discouraged because of QOC, bad performance, not actively maintained.
We recommend to use the `display` extension instead.

Reply:
replaced the `peg_oled_display` with the `display` extension